docs/api/ref/api.yml

```
@@ -427,8 +423,82 @@ paths:
             application/json:
               schema:
                 $ref: ./responses/get_preferences.yaml
+  /cgi/session.pl:
+    post:
+      summary: Login and obtain a session cookie
+      operationId: get-cgi-session.pl
+      description: Retrieve session cookie for writing operations.
+      tags:
+        - Authentication
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                user_id:
+                  type: string
+                  description: Username for login
+                password:
+                  type: string
+                  description: Password for login
+                  format: password
+              required:
+                - user_id
+                - password
+      responses:
+        "200":
+          description: Successful login
+          headers:
+            Set-Cookie:
+              schema:
+                type: string
+                description: Session cookie for subsequent authenticated requests
+              examples:
+                sessionCookie:
+                  value: "session=user123&testuser&user_session&abcdef1234567890; domain=.example.net; path=/; SameSite=Lax"
+        "401":
+          description: Authentication failed
+
 components:
+  securitySchemes:
+    cookieAuth:
+      type: apiKey
+      in: cookie
+      name: session
+      description: |
+        Session cookie containing user ID, username, and session token.
+        The value is structured as: user_id&username&user_session&session_token
+        e.g. "user_id&exampleuser&user_session&abcdefghijklmnopqrstuvwxyz123456789ABCDEFGHIJKLM"
```

For me parameters are in a different order:
```suggestion
        The value is structured as: user_session&session_token&user_id&username
        e.g. "user_session&abcdefghijklmnopqrstuvwxyz123456789ABCDEFGHIJKLM&user_id&exampleuser"
```